ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/tracing/flat/Action.java

```diff
@@ -20,8 +20,24 @@
 import org.hyperledger.besu.ethereum.api.jsonrpc.internal.results.Quantity;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 
 @JsonInclude(NON_NULL)
+@JsonPropertyOrder({
```

Comment:
I've had a lot of luck changing the order of the fields to match the desired output ordering.